src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -163,14 +187,59 @@ PS C:\> $Context = New-AzStorageContext -StorageAccountName "myaccountname" -Use
 
 This command creates a context by using the OAuth (Azure AD) Authentication.
 
+### Example 11: Create a context by specifying a storage account name, storage account key and custom blob endpoint
+```
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -StorageAccountKey "< Storage Key for MyAccountName ends with == >" -BlobEndpoint "https://MyAccountName.blob.core.windows.net/"
+```
+
+This command creates a context for the account named MyAccountName with a key for the account, and specified blob endpoint and table endpoint.
+
+### Example 12: Create a context for an anonymous storage accouont with specified file and queue endpoints 
+```
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -Anonymous -Protocol "http" -FileEndpoint "https://MyAccountName.file.core.windows.net/" -QueueEndpoint "https://MyAccountName.queue.core.windows.net/"
+```
+
+This command creates a context for anonymous use for the account named ContosoGeneral, with specified file and queue endpoints.
```

Comment:
It seems the account name in above sample is not "ContosoGeneral"

Reply:
Done